docs/logs/extending-the-sdk/Program.cs

```diff
@@ -64,6 +65,9 @@ public static void Main()
         {
             logger.LogError("{name} is broken.", "refrigerator");
         }
+
+        // message will be redacted by MyRedactionProcessor
+        logger.LogInformation("OpenTelemetry {sensitiveString}.", "<secret>");
```

Comment:
also it'd be good to modify the readme.md for the extending-the-sdk to include the redaction part. Similar to this https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/extending-the-sdk#filtering-processor